ts/parsers/colorparser.ts

```diff
@@ -17,6 +17,9 @@ export default class ColorParser {
 	}
 	public static getShapeFill(element): PowerpointElement["shape"]["fill"] {
 		//spPR takes precdence
+		if(!element["p:spPr"]) {
+			return null;
```

Comment:
fix the tab spacing on your IDE, it should be either 2 spaces or 4 spaces but not mix of both. Prefer 2 spaces

Reply:
configured it to have 120 characters per line and tab spacing to 2, and formatted all the changed scripts.